src/pages/documentation/profile/index.md

```diff
@@ -53,9 +53,3 @@ Android
 iOS
 
 <Tabs query="platform=ios&task=register"/>
-
-<!--- React Native
-
-<Tabs query="platform=react-native&task=register"/>
-
---->
```

Comment:
Oh, this comment is here for me, mostly because it makes it easier for me to add this when we add in the AEP React Native stuff.

So can you just leave this for me? Thanks.